pkg/sql/logictest/testdata/logic_test/pg_catalog

```diff
@@ -5329,3 +5329,8 @@ usename       usesuper
 regular_user  false
 root          true
 super_user    true
+
+# Testing null valid until roles
+statement ok
+CREATE USER testuser3 WITH password '123' VALID UNTIL null;
+SELECT 1 FROM pg_user WHERE usename = 'testuser3';
```

Comment:
can you actually do

```
query ... colnames
SELECT <relevant columns> FROM pg_user ...
```

to see the output is correct?

Reply:
also probably worth naming this user `regression_70180` so it's obvious what this is catching against.